integration_tests/utils.js

```diff
@@ -82,7 +82,12 @@ const cleanup = (directory: string) => rimraf.sync(directory);
 const writeFiles = (directory: string, files: {[filename: string]: string}) => {
   mkdirp.sync(directory);
   Object.keys(files).forEach(fileOrPath => {
-    const filePath = fileOrPath.split(path.sep); // ['tmp', 'a.js']
```

Comment:
can we do `const filePath = fileOrPath.split(path.posix.sep);` here? (or just `fileOrPath.split('/')`)

Reply:
Thanks for suggestion. Changed to `const filePath = fileOrPath.split('/')`